### PR TITLE
dashboard: Break out dashboard manifest into its own component

### PR DESCRIPTION
**Pull request by stefwalter**
This way it can be enabled/disabled independently.

### CODE REVIEW

File: tools/cockpit.spec [hunk 1]
```diff
@@ -207,7 +207,10 @@ install -p -m 644 cockpit.pp %{buildroot}%{_datadir}/selinux/targeted/
 echo '%dir %{_datadir}/%{name}/base1' > shell.list
 find %{buildroot}%{_datadir}/%{name}/base1 -type f >> shell.list
 
-echo '%dir %{_datadir}/%{name}/shell' >> domain.list
+echo '%dir %{_datadir}/%{name}/dashboard' >> shell.list
+find %{buildroot}%{_datadir}/%{name}/dashboard -type f >> shell.list
+
+echo '%dir %{_datadir}/%{name}/domain' >> domain.list
```
Comment (mvollmer) [inline tools/cockpit.spec]: `echo '%dir %{_datadir}/%{name}/domain' >> shell.list`?

Comment (stefwalter) [inline tools/cockpit.spec]: Thanks. Good catch.

**Comment by stefwalter** [comment]
Rebased and fixed for review.

**Comment by stefwalter** [comment]
@mvollmer @dperpeet Why is r7 hubbot 'sad'? When I click on 'Details' i see 'VERIFICATION PASSED' here: http://files.cockpit-project.org/hubbot/85302333ef451d31b0d4ee32e1a01ed80a456eaf.1/hubbot.html

**Comment by mvollmer** [comment]
> Why is r7 hubbot 'sad'? 

Hmm, the "Details" link of r7 actually goes to the results of the f22 run.  What might have happened is that the r7 result was deleted (because of age) and then the f22 run picked the same directory name.

One solution is to use a simple GC to reclaim disk space, using the "Details" links as the roots, instead of the current "keep 30 directories" rule.

**Comment by dperpeet** [comment]
> Hmm, the "Details" link of r7 actually goes to the results of the f22 run. What might have happened is that the r7 result was deleted (because of age) and then the f22 run picked the same directory name.
> 
> One solution is to use a simple GC to reclaim disk space, using the "Details" links as the roots, instead of the current "keep 30 directories" rule.

Yes, I'm pretty sure that happened. It would be good to actually write the current build info out first. Sadly, updating our CI system isn't a high priority right now...

**Pull request closed without merging**